### PR TITLE
New client

### PR DESCRIPTION
**Pull request by musketyr**
using new Java Sentry client

### CODE REVIEW

File: src/test/groovy/grails/plugin/sentry/SanityInegrationSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package grails.plugin.sentry
+
+import grails.test.mixin.integration.Integration
+import spock.lang.Specification
+
+@Integration
+class SanityInegrationSpec extends Specification {
```
Comment (benorama) [inline src/test/groovy/grails/plugin/sentry/SanityInegrationSpec.groovy]: You mean `SanityIntegrationSpec` ?

Comment (musketyr) [inline src/test/groovy/grails/plugin/sentry/SanityInegrationSpec.groovy]: of course! should be ok now.

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ Declare the plugin dependency in the _build.gradle_ file, as shown here:
 ```groovy
 dependencies {
     ...
-    compile("org.grails.plugins:sentry:7.8.1")
+    compile("org.grails.plugins:sentry:Sentry-1.2.0")
```
Comment (benorama) [inline README.md]: I find it strange to add`Sentry-` to version number.
If want to keep version number, we could prepend `1` to sentry client, so next release would be `11.2.0`.
What do you think?

Comment (musketyr) [inline README.md]: sounds good. I wan't sure about the versioning as well.

Comment (musketyr) [inline README.md]: done

**Pull request merged**